composer/algorithms/mixup/mixup.py

```diff
@@ -64,7 +64,9 @@ def mixup_batch(input: torch.Tensor,
             X = torch.randn(N, C, H, W)
             y = torch.randint(num_classes, size=(N,))
             X_mixed, y_perm, mixing = mixup_batch(
-                X, y, alpha=0.2)
+                X, y,
+                alpha=0.2
+            )
```

Comment:
This looks a bit funny to me. Is this how we typically format things here?

Reply:
yeah it's usually:
```
X, 
y,
alpha=0.2
```

Reply:
My bad, just fixed it.